docs/platform/cloud/managing-airbyte-cloud/manage-airbyte-cloud-notifications.md

```diff
@@ -4,61 +4,66 @@ products: all
+- Self-Managed versions of Airbyte can send notifications to a webhook, but not an email.
 
-To edit this setting, click **Connections** and select the connection you want to receive notifications for. Click the **Settings** tab on the Connection page. In the **Advanced Settings**, toggle **Schema update notifications**.
+### Events Airbyte can notify you about
+
+| Event                                   | Cloud    | Self-Managed | Description                                                                                                                                            |
+| --------------------------------------- | -------- | ------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| **Failed syncs**                        | &#10003; | &#10003;     | A sync from any of your connections fails. If syncs runs frequently or if there are many syncs in the workspace, this event can be noisy.              |
+| **Successful syncs**                    | &#10003; | &#10003;     | A sync from any of your connections succeeds. If syncs runs frequently or if there are many syncs in the workspace, this event can be noisy.           |
+| **Connection updates**                  | &#10003; | &#10003;     | A connection is updated automatically. For example, a source schema is automatically updated.                                                          |
+| **Connection updates requiring action** | &#10003; | &#10003;     | A connection update requires you to take action. For example, a breaking change.                                                                       |
+| **Warning - repeated failures**         | &#10003; | &#10003;     | Airbyte is at risk of turning off a connection due to repeated failures. It has failed 20 times consecutively and his not been successful in the last 4 days. |
+| **Sync disabled - repeated failures**   | &#10003; | &#10003;     | Airbyte has turned off a connection due to repeated failures. It has failed 30 times consecutively and has not been successful in the last 7 days.     |
+| **Warning - upgrade required**          | &#10003; |              | A new connector version is available, but you need to manually upgrade.                                                                                |
+| **Sync Disabled - upgrade required**    | &#10003; |              | Airbyte turned off one or more connections automatically because you missed the deadline to upgrade the connector.                                     |
 
-## Configure Email Notification Settings
+### Enabling schema update notifications
 
-<AppliesTo cloud />
+If you want Airbyte to notify you of source schema changes, enable the following notifications.
 
-To set up email notifications, click **Settings** and navigate to **Workspace** > **Notifications**.
+- `Automatic Connection Updates`
```

Comment:
I think this should be `Connection updates`, also the capitalization in the table vs. the items in this list is inconsistent. In the UI we capitalize all words, e.g. `Connection Updates Requiring Action`